internal/provisioner/objects/deployment/deployment.go

```diff
@@ -97,7 +97,9 @@ func DesiredDeployment(contour *model.Contour, image string) *appsv1.Deployment
 		fmt.Sprintf("--contour-config-name=%s", contour.ContourConfigurationName()),
 		fmt.Sprintf("--leader-election-resource-name=%s", contour.LeaderElectionLeaseName()),
 		fmt.Sprintf("--envoy-service-name=%s", contour.EnvoyServiceName()),
+		fmt.Sprintf("--kubernetes-debug=%d", contour.Spec.KubernetesLogLevel),
```

Comment:
Please add some unit test cases for this.

Reply:
done